README.md

```diff
@@ -13,10 +13,11 @@ The project is live [here](https://rest-fetch.herokuapp.com/)
 ## Installation
 Follow the instructions to get started with the project on your local machine 🚀
 * Fork and clone the project
-* `cd fetch/` and run `npm install` to install dependencies
```

Comment:
We should not remove the non-docker method completely. Instead, write a separate section in README for the Docker setup.

Reply:
Okay, will see this